addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientListScreen.js

```diff
@@ -84,11 +86,11 @@ odoo.define('point_of_sale.ClientListScreen', function(require) {
          */
         get nextButton() {
             if (!this.props.client) {
-                return { command: 'set', text: 'Set Customer' };
+                return { command: 'set', text: _t('Set Customer') };
```

Comment:
`_t` --> `this.env._t` (and don't need const core = require('web.core'); const _t = core._t;)

Reply:
thanks !